src/Sulu/Bundle/AdminBundle/Resources/js/components/RectangleSelection/modifiableRectangle.scss

```diff
@@ -0,0 +1,37 @@
+@import '../../containers/Application/colors.scss';
+
+$backdropColor: rgba($black, 0.6);
+$borderColor: $white;
+$borderSize: 2px;
+
+.rectangle {
+    position: absolute;
+    background: transparent;
+    border: $borderSize solid $borderColor;
+    cursor: move;
+    overflow: visible;
+    z-index: 1;
```

Comment:
Didn't you read the [comment on css-tricks](https://css-tricks.com/handling-z-index/#comment-1580226) I've added before? That way it would be super easy to insert new z-index values. You would just have to add both `z-index`es used in here to a variable map.